cocoa/events_darwin.go

```diff
@@ -46,6 +47,10 @@ func (w *Window) EventChan() (events <-chan interface{}) {
 	downKeys := make(map[string]bool)
 	ec := make(chan interface{})
 	go func(ec chan<- interface{}) {
+
+		noX := -1
+		lastXY := image.Point{-1, -1}
```

Comment:
Or, 
var noPoint = image.Point{-1, -1}
var lastXY = noPoint

and compare lastXY to noPoint instead of lastXY.X to noX